docs/reference/commandline/dockerd.md

```diff
@@ -1463,6 +1468,9 @@ This is a full example of the allowed configuration options on Windows:
   "bridge": "",
   "cluster-advertise": "",
   "cluster-store": "",
+  "containerd": "\\\\.\\pipe\\containerd-containerd",
```

Comment:
I'm not _100%_ sure this is correct, but I would say I'm about 99% sure now (with @cpuguy83's help digging :heart:).

cc @StefanScherer perhaps you can help confirm? :innocent: :pray: :heart:

Reply:
(This gets passed directly to `containerd.New`, which on Linux just takes the direct socket path not a URI like `unix://...`)

Reply:
The UNC path looks good for the named pipe, otherwise a URI would look like `npipe://...`